HW8/MariiaDiachuk/08_Task_3/main.py

```
@@ -0,0 +1,33 @@
+from shapes import rectangle_area, triangle_area, circle_area
+import math
```

Ruff (0.8.2)

`math` imported but unused

Remove unused import: `math`

(F401)